src/Jackett.Common/Indexers/BJShare.cs

```diff
@@ -117,28 +117,48 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
 
         private string InternationalTitle(string title)
         {
-            // Get international title if available, or use the full title if not
-            var cleanTitle = Regex.Replace(title, @".* \[(.*?)\](.*)", "$1$2");
-            cleanTitle = Regex.Replace(cleanTitle, @"(?:.*)\/(.*)", "$1");
-
-            return cleanTitle.Trim();
+            var match = Regex.Match(title, @".* \[(.*?)\]$");
+            return match.Success ? match.Groups[1].Value : title;
         }
         private string StripSearchString(string term, bool isAnime)
         {
             // Search does not support searching with episode numbers so strip it if we have one
             // Ww AND filter the result later to archive the proper result
-            term = Regex.Replace(term, @"[S|E]\d\d", string.Empty).Trim();
-            return isAnime ? term.TrimEnd(_digits) : term;
+            term = _EpisodeRegex.Replace(term, string.Empty);
+            term = isAnime ? Regex.Replace(term, @"\d*$", string.Empty) : term;
+            return term.TrimEnd();
```

Comment:
This seems like it changes functionality compared to the original code. 
```suggestion
            term = _EpisodeRegex.Replace(term, string.Empty).Trim();
            term = isAnime ? Regex.Replace(term, @"\d*$", string.Empty).TrimEnd() : term;
            return term;
```

Reply:
It was just simplified, the result is still the same, tested it